phpstan-baseline.neon

```diff
@@ -1,5 +1,168 @@
 parameters:
 	ignoreErrors:
+		-
+		    # will be fixed in v4. Currently BC break
```

Comment:
Please don't mix tabs and spaces

Reply:
I thought that PhpStorm will handle this thing